Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1762,22 +1762,34 @@ namespace Remotion.BocAutoCompleteReferenceValue
         const position = calculateSpaceAround(reference);
 
         const isVisibe = LayoutUtility.IsVisible(popUp);
+        let scrollbarGutterValue;
         if (!isVisibe)
         {
             popUp.style.width = 'auto'; // clear the width before showing the popUp, otherwise, the popUp expands to 100%
+            scrollbarGutterValue = popUp.style.getPropertyValue("scrollbar-gutter");
+            popUp.style.setProperty("scrollbar-gutter", "stable") //Typescript build error when using 'popUp.style.scrollbarGutter = "stable";'
```

Comment:
```suggestion
            popUp.style.setProperty("scrollbar-gutter", "stable");
```
also noticed the missing `;`